image/base/Dockerfile

```diff
@@ -81,10 +81,10 @@ RUN add-apt-repository ppa:ubuntu-toolchain-r/test &&\
     echo 'gem: --no-document' >> /usr/local/etc/gemrc &&\
     mkdir /src && cd /src && git clone https://github.com/sstephenson/ruby-build.git &&\
     cd /src/ruby-build && ./install.sh &&\
-    cd / && rm -rf /src/ruby-build && ruby-build 2.0.0-p647 /usr/local &&\
+    cd / && rm -rf /src/ruby-build && ruby-build 2.3.1 /usr/local &&\
     gem update --system &&\
     gem install bundler &&\
-    rm -rf /usr/local/share/ri/2.0.0/system &&\
```

Comment:
`lsb-releases` does not come installed with the base image.

`sudo` is required before https://github.com/tgxworld/discourse_docker/blob/ddf77f732f8a9fbd5c048c531a5fbc942e7438b5/image/base/Dockerfile#L27